optuna/cli.py

```diff
@@ -369,6 +373,122 @@ def take_action(self, parsed_args: Namespace) -> None:
             )
 
 
+class _Ask(_BaseCommand):
+    """Create a new trial and suggest parameters."""
+
+    def get_parser(self, prog_name: str) -> ArgumentParser:
+
+        parser = super(_Ask, self).get_parser(prog_name)
+        parser.add_argument("--study-name", type=str)
+        parser.add_argument("--direction", type=str, choices=("minimize", "maximize"))
+        parser.add_argument("--directions", type=str, nargs="+", choices=("minimize", "maximize"))
+        parser.add_argument("--sampler", type=str)
+        parser.add_argument("--sampler-kwargs", type=str)
+        parser.add_argument("--search-space", type=str)
```

Comment:
I want to know why `search-space` is not required.

Reply:
Good point. The reason is that I tried to keep behaviors consistent. It's allowed for a trial not to be associated with any parameters. I.e. it's fine to not call `Trial.suggest_` in an objective function, and requiring a search space here I though would be inconsistent with this.

Reply:
Thank you for the clarifying. I undepstand and agree with you to keep a consistency.

Reply:
By the way, we could require the argument, forcing the user to pass an empty dictionary to emulate the above situation, but I'm not sure if that'd make any users happy.

Reply:
I also think it doesn't make users happy. The current implementation feels good to me.